llisite/urls.py

```diff
@@ -5,11 +5,12 @@
 from django.conf import settings
 
 urlpatterns = [
-    # Examples:
-    # url(r'^$', 'llisite.views.home', name='home'),
-    # url(r'^blog/', include('blog.urls')),
-
     url(r'^admin/', include(admin.site.urls)),
+<<<<<<< HEAD
```

Comment:
@JiwonHwang  I think we forgot to add this file after resolving the conflict.  Please just add & commit this file and then push.